.travis.yml

```diff
@@ -17,6 +15,5 @@ script:
   - yarn test:a11y
   - yarn test:nsp
   - yarn test:coverage
-after_script: greenkeeper-lockfile-upload
 after_success:
```

Comment:
What is the reason we remove greenkeeper from ccd-admin-web.  This is the only repo that greenkeeper got in.